src/epic.actions.js

```diff
@@ -16,7 +16,12 @@ async function getEpic({ id }) {
 
 async function describeEpic({ id }) {
   const { epics } = await getEpic({ id });
-  const stories = (await makeGetRequest(`epic/${ id }/issue`)).issues;
+  const epicIssues = await makeGetRequest(`epic/${id}/issue`);
```

Comment:
nit & definitely not blocking, but you could do these 2 requests in parallel since they're not dependent on each other:

```diff
-  const { epics } = await getEpic({ id });
-  const epicIssues = await makeGetRequest(`epic/${id}/issue`);
+  const [{ epics }, epicIssues] = await Promise.all([
+    getEpic({ id }),
+    makeGetRequest(`epic/${id}/issue`)]);
```